### PR TITLE
Kepler Interface up with Next.js, but still blank map

### PR DESCRIPTION
**Pull request by justin-kleid**

### CODE REVIEW

File: .env.local
Comment (lixun910) [inline .env.local:1]: Thanks @justin-kleid ! 

Suggestion: don't commit this file which contains credential information. This should be on your local machine. You can add it to .gitignore. 
When we need to create github actions to build the package, we can store the token in github's Repository secrets.

File: .DS_Store
Comment (lixun910) [inline .DS_Store:1]: Suggestion: ignore this file in .gitignore

File: public/favicon.ico
Comment (lixun910) [inline public/favicon.ico:1]: suggestion: we can remove this file and replace it with ours later.

File: public/vercel.svg
Comment (lixun910) [inline public/vercel.svg:1]: suggestion: we can remove this file and replace it with ours later.

File: styles/Home.module.css
Comment (lixun910) [inline styles/Home.module.css:1]: suggestion: we can clean up this file and remove the CSS styles that we don't actually use.

**Comment by justin-kleid** [comment]
Got it - will keep in mind. Thanks!

**Comment by lixun910** [comment]
@justin-kleid You can merge this PR once the comments are addressed. Thanks!

**Comment by justin-kleid** [comment]
Will do. Should have it cleaned up before Thursday.

File: pages/Map.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import KeplerGl from "kepler.gl";
+import {ReactReduxContext} from "react-redux";
+
+const MAPBOX_TOKEN = process.env.MapboxAccessToken;
```
Comment (lixun910) [inline pages/Map.jsx:5]:
```suggestion
const MAPBOX_TOKEN = process.env.NEXT_PUBLIC_MAPBOX_API_TOKEN;
```

File: next.config.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
```
Comment (lixun910) [inline next.config.js:3]:
```suggestion
  reactStrictMode: false,
```

**Comment by lixun910** [comment]
@justin-kleid Hi Justin, can you merge this one PR first? You will see another pull request https://github.com/GeoDaCenter/reactgeoda/pull/4 has only two more commits to merge, which is cleaner. Thanks!

**Review by lixun910 — APPROVED** [review]

**Comment by justin-kleid** [comment]
Think I just did, but let me know if went through how it should have.

**Comment by justin-kleid** [comment]
Apologies for the disorganization in my git workflow.

**Pull request merged**